docs/docfiles/intro.rst

```diff
@@ -1,17 +1,31 @@
 Welcome to Data-parallel Control (dpctl)'s documentation!
 =========================================================
 
-The data-parallel control (dpctl) library provides C and Python bindings for
+`Data-parallel control <https://github.com/IntelPython/dpctl>`_ (dpctl) is a
+runtime library for Python applications and libraries to execute a compute
+kernel on a device that supports a data-parallel mode of execution. Using
+dpctl's API a library or an application can query a system to identify
+data-parallel devices, allocate memory on those devices, and schedule execution
+of compute kernels on the devices. Dpctl's role is only to facilitate the
+scheduling of compute kernels, the library plays no role in
+the definition of the kernels themselves. It is up to the users of dpctl to
+define the kernels. As an example, the
+`numba-dppy <https://intelpython.github.io/numba-dppy/latest/index.html>`_
+package uses an OpenCL-like abstraction to define kernels directly in Python
+and JIT compiles them to native binary. Numba-dppy then schedules the kernels on
+devices using dpctl. Another example is the
+`dpnp <https://intelpython.github.io/dpnp/>`_ package, a NumPy-like
+library of pre-compiled kernels written in the
+`SYCL language <https://sycl.readthedocs.io/en/latest/index.html>`_. Dpnp too
+uses dpctl to schedule and execute the kernels it provides.
```

Comment:
>  Dpnp too uses dpctl to schedule and execute the kernels it provides.

I do not think this is true. `dpnp` only uses `dpctl.SyclQueue` provided by the user, and submits to that queue by directly calling the DPC++ run-time, i.e. `q.submit(...)`.